massgov/mayflower
# DP-10665 [a11y: search.mass.gov] The state seal in the footer

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

The state seal image in the footer had unnecessary text alternative information even it was used for a decorative purpose.  It was not useful to AT users.

The mark up is changed to match the Mass.gov site's:

- A link to the Mass.gov home is added to the image to be more functional.
- The alt value of the state seal image is removed since it's deprecate with the title attribute in its parent link.

## Related Issue / Ticket

- [DP-10665 \[a11y: search.mass.gov\] Footer logo with alt value, Massachusetts State Seal.](https://jira.mass.gov/browse/DP-10665)

## Steps to Test


1. `npm install && npm start`
2. Check the footer state seal for:
       - the image is linked to the Mass.gov home (http://www.mass.gov).
       - the link has title attribute with its value: Mass.gov homepage.
       - the image has `alt` attribute, but its value is empty.

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
